README.md

```diff
@@ -43,7 +43,17 @@ Want to get paid for your contributions to `one-app`?
 
 ### Quick Start
 
-**Clone and Install One App:**
+**Build a Module with [generator-one-app-module](https://github.com/americanexpress/one-app-cli/tree/master/packages/generator-one-app-module)**
+
+The easiest way to do this is via [npx](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner) (comes with `npm` versions 5.2.0 and above). Run the following command in the directory you want your module to live:
+
+```bash
+npx -p yo -p @americanexpress/generator-one-app-module -- yo one-app-module
+```
+
+This will kick off the yeoman generator to scaffold your module.
```

Comment:
Can you link to the generator's README so user's can get more information if they'd like?

Reply:
And probably don't need to mention that it is a yeoman generator, this could confuse users.

Reply:
How about `This will use the [one-app-module-generator](LINK) to generate a basic One App module.`

Reply:
fixed 030f665